src/transformers/models/bart/modeling_bart.py

```diff
@@ -148,7 +276,9 @@ def __init__(
         num_heads: int,
         dropout: float = 0.0,
         is_decoder: bool = False,
+        is_causal: bool = False,
```

Comment:
Personal taste, but I would add this arg after `bias` in case somebody is using positional arguments.